src/com/projectkorra/projectkorra/airbending/flight/FlightMultiAbility.java

```diff
@@ -227,14 +228,15 @@ public void run() {
 						if (!GeneralMethods.isRegionProtectedFromBuild(this.player, e.getLocation())) {
 							final LivingEntity le = (LivingEntity) e;
 							DamageHandler.damageEntity(le, this.speed / 2, this);
-							le.setVelocity(this.player.getVelocity().clone().multiply(2 / 3));
+							GeneralMethods.setVelocity(this,le, this.player.getVelocity().clone().multiply(2 / 3));
 						}
 					}
 				}
 			}
 
 			this.particles();
-			this.player.setVelocity(this.player.getEyeLocation().getDirection().clone().multiply(this.multiplier));
+			GeneralMethods.setVelocity(this,this.player, this.player.getEyeLocation().getDirection().clone().multiply(this.multiplier));
+			
```

Comment:
We don't need the white-space under this line. Additionally, please change to
`GeneralMethods.setVelocity(this, this.player, this.player.getEyeLocation().getDirection().clone().multiply(this.multiplier));`